plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -10,10 +10,15 @@ class Release < ActiveRecord::Base
 
     validates :release_group, presence: true
     validates :deploy_group, presence: true
-    validates :status, inclusion: VALID_STATUSES
+    validates :status, inclusion: STATUSES
 
     after_initialize :set_default_status, on: :create
 
+    def release_is_live
+      self.status = 'live'
+      self.deploy_finished_at = Time.now
```

Comment:
Nevermind, I just checked that, yes, it is required when calling a self write accessor. My java background was playing tricks to me.